File: CurrentDate/app/src/main/java/com/example/currentdate/MainActivity.java
package com.example.currentdate;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

//import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Date;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Calendar calendar = Calendar.getInstance();
        String currentDate = DateFormat.getDateInstance(DateFormat.FULL).format(calendar.getTime());

        Date currentTime = new Date();
        SimpleDateFormat timeFormat = new SimpleDateFormat("HH:mm:ss");
        String timeString = timeFormat.format(currentTime);


        TextView textViewDate = findViewById(R.id.text_view_date);
        textViewDate.setText("Abhay Dayma\n"+currentDate+"\n" + timeString);
    }
}

File: Task ButtonSound/ButtonSound/app/src/main/java/com/example/buttonsound/MainActivity.kt
package com.example.buttonsound

import android.R
import android.media.MediaPlayer
import android.media.MediaPlayer.OnCompletionListener
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    var player: MediaPlayer? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun play(v: View?) {
        if (player == null) {
            player = MediaPlayer.create(this, R.drawable.song2)
            player!!.setOnCompletionListener(OnCompletionListener { stopPlayer() })
        }
        player!!.start()
    }

    fun pause(v: View?) {
        if (player != null) {
            player!!.pause()
        }
    }

    fun stop(v: View?) {
        stopPlayer()
    }

    private fun stopPlayer() {
        if (player != null) {
            player!!.release()
            player = null
            Toast.makeText(this, "MediaPlayer released", Toast.LENGTH_SHORT).show()
        }
    }

    override fun onStop() {
        super.onStop()
        stopPlayer()
    }
}